lib/js/components/Banner/Banner.stories.mdx

```diff
@@ -43,7 +45,7 @@ import { ICONS } from '../Icon';
 		},
 		buttonText: {
 			control: { type: 'text' },
-			defaultValue: null,
+			defaultValue: '',
```

Comment:
this prop is still rendered with red border. I think you have to update line 23.

Reply:
Works for me: 
![image](https://user-images.githubusercontent.com/7030884/180218050-e943d259-9f18-4119-94c5-87fbe9b3697e.png)

Reply:
Actually, you were right, fixed in https://github.com/bethinkpl/design-system/commit/51bf6610124fc787057cd845103342d4b79381bf